RetroBar/Languages/Tiếng Việt.xaml

```diff
@@ -39,7 +39,7 @@
     <s:String x:Key="behavior_heading">Hành vi</s:String>
 
     <s:String x:Key="start_text">Bắt đầu</s:String>
-    <s:String x:Key="start_text_xp">Bắt đầu</s:String>
+    <s:String x:Key="start_text_xp">bắt đầu</s:String>
```

Comment:
In Vista, the XP Start button appears in lowercase, but in XP, it is the other way around:
https://youtu.be/ZEAJ0_VjmL8?t=143

Reply:
thanks i got it